dts/msm8953-huawei-milan.dts

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2023, bzy-080408<Bzy080408@outlook.com>
+
+/dts-v1/;
+
+/include/ "msm8953.dtsi"
+
+/ {
+	model = "Huawei Maimang 5 / Nova (Plus) / G9 (Plus)";
+	compatible = "huawei,sdm625", "qcom,msm8953", "lk2nd,device";
+	qcom,msm-id = <293 0x0>;
+	qcom,board-id = <8000 0>, <8001 0>, <8002 0>, <8003 0>, <8016 0>,
+			<8017 0>, <8018 0>, <8019 0>, <8032 0>, <8033 0>,
+			<8035 0>, <8048 0>, <8049 0>, <8050 0>, <8051 0>,
+			<8066 0>, <8064 0>, <8067 0>, <8080 0>, <8082 0>,
+			<8083 0>, <8098 0>, <8099 0>, <8114 0>, <8130 0>;
+
+	panel {
+		compatible = "huawei,sdm625-panel";
```

Comment:
dunno what convention we use for panel compatibles, we might want to change that

Reply:
changed it to `huawei,milan-panel` to align with other devices